src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
 spring.application.name=GitHubAiAPI
```

Comment:
The property 'spring.application.name' is missing a value. Please provide a value for it.